config/prow/config_knative.yaml

```diff
@@ -402,6 +402,16 @@ presubmits:
     - integration-tests: false
     - go-coverage: false
 
+  knative-sandbox/operator:
+    - build-tests: true
+      dot-dev: true
```

Comment:
Do projects in this org also use the `knative.dev` path alias for dep? If not, please remove all `dot-dev: true` settings.

Reply:
We haven't figured this out yet, so I'll remove it.